docs/tutorials/coin_spend_rpc.md

```diff
@@ -0,0 +1,70 @@
+---
+id: coin_spend_rpc
+title: How to spend a coin using an RPC call
+sidebar_label: Spend a coin using RPC
+---
+
+This tutorial teaches you how to spend a coin with any puzzle using RPC call. I'm using the password-locked coin puzzle from [How to secure coin with a custom puzzle](custom_puzzle_secure) tutorial in this tutorial as well.
```

Comment:
"using RPC calls".  Also, make sure that you use terms like "we" and "our" instead of "I" and "my".